src/theme/variables/edgeLight.js

```diff
@@ -186,6 +186,17 @@ export const edgeLight: Theme = {
 
   // Misc
   // pressedOpacity: 0.25, // Should be removed when press colors are given to buttons and links
+  modalIcon: palette.edgeBlue,
+
+  // Scene/Components background
+  promoCardBackgroud: palette.edgeBlue,
```

Comment:
I don't think any of these actually had to be added. All of them represent usage of the tile background color that is similar to the actual tile component. All of these things will be re-factored anyway so for now just have them use tileBackground which is transparent

Reply:
removed, though did retain walletListBackground because being transparent will always expose the underlaying swipe display